CombineDemo/CombineDemo/PokemonApi/PokemonApiEndpoint.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PokemonApiEndpoint.swift
+//  CombineDemo
+//
+//  Created by Manish Rathi on 19/08/2023.
+//
+
+import Foundation
+
+enum PokemonApiEndpoint: ApiEndpointType {
```

Comment:
Nooo!
It should be `struct`

Reply:
You no need the protocol. Just make the base type and use static let for specific endpoints

Reply:
Thank you, i agree!!
- Yes, it can be indeed a struct, but i just added an enum for these simple 3 endpoints. 
- Also added protocol **for Mocks**, for better unit-testing (please see attached)

<img width="598" alt="Screenshot 2023-08-21 at 08 15 48" src="https://github.com/crazymanish/what-matters-most/assets/5364500/244a8508-2393-480a-ad2b-ee9de7eb5aa4">

Reply:
You can mock struct as you want as well